apps/codeforafrica/src/components/Footer/Footer.js

```diff
@@ -22,12 +23,24 @@ const FooterRoot = styled(Box)(
 );
 
 const Footer = React.forwardRef(function Footer(props, ref) {
-  const { subscription } = props;
+  const { subscription, menu } = props;
 
   return (
     <FooterRoot ref={ref}>
       <Section sx={{ px: { xs: "20px", sm: 0 } }}>
         <Grid container justifyContent="flex-end">
+          <Grid
+            item
+            xs={12}
+            md={3}
+            sx={{
+              order: { xs: 1, md: 0 },
+              display: "flex",
+              justifyContent: "center",
```

Comment:
```suggestion
```
These can be set in the `FooterNavList` component since it's specifically for the footer.

Reply:
@kilemensi Returned the Grid back here following this comment https://github.com/CodeForAfrica/ui/pull/50#discussion_r866862986